rules/falco_rules.yaml

```diff
@@ -921,6 +921,10 @@
               container.image.repository endswith "openshift3/ose-logging-fluentd" or
               container.image.repository endswith "containernetworking/azure-npm")
 
+- macro: containerd_activities
+  condition: proc.name=containerd and (fd.name startswith "/var/lib/containerd/io.containerd.snapshotter.v1.overlayfs/snapshots/" or
```

Comment:
Just a super minor change request:

Add parentheses to be future proof if we were to keep adding on e.g. `and not (containerd_activities or other_macro)`

```
(proc.name=containerd and (fd.name startswith "/var/lib/containerd/io.containerd.snapshotter.v1.overlayfs/snapshots/" or
                                       fd.name startswith "/var/lib/containerd/tmpmounts/"))
```